ExternalLibs/trimesh2/include/map.h

```diff
@@ -48,8 +48,8 @@ struct map
         typedef MapFilter<FType,InType,OutType> MF;
         typename MF::Pointer mf = MF::New();
         mf->SetFunctor(&functor); // handles output
-        if(numThreads>0)
-            mf->SetNumberOfThreads(numThreads);
+        // if(numThreads>0)
```

Comment:
I saw some discussion about this on issue #89, but I'm not sure what this has to do with compile time warnings.  

This doesn't change the behavior at all?

Reply:
It won't change behavior, though it could slow some computation down. But I'm not even sure if this code is used, and it accounts for half of the warnings when we compile. Those warnings hide other important warnings -- there are so many I can't even find build errors when using Xcode! 

I commented on #89 in association with this commit yesterday (with a link to how to upgrade this deprecated multithread functionality) because that issue is the reason we still have this ancient code in the first place. This piece of our version of trimesh may not even be part of trimesh (I couldn't find it anywhere in the latest version).

Reply:
@prafulag, @sheryjoe can you comment on this section of code?  I'm all for reducing compile warnings, but I'm not sure we should trade execution time for it, if that is truly the case. (I have no idea)